components/director/internal/open_resource_discovery/validation_test.go

```diff
@@ -1587,6 +1611,33 @@ func TestDocuments_ValidateBundle(t *testing.T) {
 				doc := fixORDDocument()
 				doc.ConsumptionBundles[0].CredentialExchangeStrategies = json.RawMessage("[]")
 
+				return []*ord.Document{doc}
+			},
+		},
+		{
+			Name: "Invalid `correlationIds` field when it is invalid JSON for Bundle",
```

Comment:
Will you add one more test where a valid JSON is passed but the value of one of the correlationIDs is invalid (does not match the required regex)